application/src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -134,7 +137,9 @@ public void buildGraph() {
         /* Make transfers to each nearby stop that has lowest weight on some trip pattern.
          * Use map based on the list of edges, so that only distinct transfers are stored. */
         Map<TransferKey, PathTransfer> distinctTransfers = new HashMap<>();
-        RegularStop stop = ts0.getStop();
+        RegularStop stop = Objects.requireNonNull(
+          timetableRepository.getSiteRepository().getRegularStop(ts0.getId())
```

Comment:
would it make sense to rename the existing method SiteRepository.getRegularStop() to findRegularStop() and introduce a new method getRegularStop() that performs the null check?

Reply:
Well, according to our [naming conventions](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/doc/dev/decisionrecords/NamingConventions.md) `findXXX` methods should return `Optional` or a collection. For that reason I'm hesitant. I totally support this idea though.